poker/utils/message.py

```diff
@@ -0,0 +1,148 @@
+            raise NegativeException
+        if player_response >= chip_count:
+            raise InsufficientChipException
+        return player_response
+    
+    def different_amount(self, chip_count: int) -> None:
+        print(f"You only have {chip_count} chips. Please select a lower amount.")
+    
+    def player_summary(self, players: dict) -> None:
+        player_table = Table(title="Player Summary")
+        player_table.add_column("Order")
+        player_table.add_column("Name")
+        player_table.add_column("Chips")
+        player_table.add_column("Blind")
+        player_table.add_column("Pocket Cards")
+        player_table.add_column("Best Hand")
+        player_table.add_column("Short Name")
+        for player_id, player in players.items():
+            player = player["player"]
+            player_table.add_row(
+                str(player_id), 
+                player.name, 
+                f"{GameStack.WHITE['name']}: {player.stack.chips[GameStack.WHITE['name']]}",
+                "Big" if player_id == 1 else "Small",
+                " ".join(f"{card}" for card in player.pocket_cards) if player.name == PLAYER_NAME or len(player.best_hand) > 1 else "Hidden",
+                ", ".join(f"{card}" for card in player.best_hand["hand"]) if len(player.best_hand) > 1 else "",
+                player.best_hand["short"] if len(player.best_hand) > 1 else ""
+                )
+        console = Console()
+        console.print("", player_table)
+    
+    def game_progression_prompt(self, progress: bool = None) -> None:
+        try:
+            if progress:
+                player_response = input(f"How about now, are you ready? [yes/no] ").lower()
+            else:
+                player_response = input(f"Are you ready to continue? [yes/no] ").lower()
+            if player_response not in self.decision:
+                raise GamePlayException
+            if player_response in [Decision.N.value, Decision.NO.value]:
+                raise NotReadyException
+        except GamePlayException:
+            sleep(GAME_DELAY)
+            self.game_progression_prompt()
+        except NotReadyException:
+            sleep(GAME_DELAY)
+            self.game_progression_prompt(progress=True)    
+
+    def game_summary(self, pot: GameStack, community_cards: list) -> None:
+        game_pot = [Panel(f"Game Pot\n{key}: {value}") for key, value in pot.chips.items()]
+        game_pot.extend(Panel(f"Card {card_number + 1}\n{community_cards[card_number]}") for card_number in range(len(community_cards)))
+        console = Console()
+        console.print(Columns(game_pot))
+
+    def showdown(self, winner: dict, pot: GameStack, players: dict, community_cards: list[Card]) -> None:
+        sleep(GAME_DELAY)
+        self.game_summary(pot=pot, community_cards=community_cards)
+        sleep(GAME_DELAY)
+        self.player_summary(players=players)
+        sleep(GAME_DELAY)
+
+        if len(winner) > 3:
```

Comment:
Ideally you put magic numbers into constants as well

Reply:
ah, I overlooked both of these. Thanks.